placenames.mss

```diff
@@ -84,51 +84,57 @@
 
 #placenames-medium::town {
   [place = 'town'] {
-    [zoom >= 9] {
+    [zoom >= 9][zoom < 16] {
       text-name: "[name]";
       text-size: 9;
       text-fill: @placenames;
       text-face-name: @book-fonts;
       text-halo-radius: 1.5;
       text-halo-fill: rgba(255,255,255,0.6);
-      text-wrap-width: 50;
+      text-wrap-width: 45;
       text-min-distance: 10;
-    }
-    [zoom >= 11] {
-      text-size: 11;
-      text-wrap-width: 70;
-    }
-    [zoom >= 14] {
-      text-size: 15;
-      text-wrap-width: 90;
-      text-fill: @placenames-light;
-      text-halo-fill: white;
+      [zoom >= 11] {
+        text-size: 11;
+        text-wrap-width: 55;
+      }
+      [zoom >= 12] {
+        text-size: 13;
+        text-wrap-width: 65;
+      }
+      [zoom >= 14] {
+        text-size: 15;
+        text-wrap-width: 75;
+      }
     }
   }
 }
 
 #placenames-small::suburb {
```

Comment:
We probably want to move these out of their own attachments and do an order by on the query - but out of scope of this PR